rotkehlchen/ethchain.py

```diff
@@ -76,6 +102,19 @@ def set_rpc_port(self, port: int) -> Tuple[bool, str]:
             self.ethrpc_port = port
         return result, message
 
+    def get_eth_highest_block(self):
+        """ Attempts to query blockcypher for the block height
+
+        Returns the highest blockNumber"""
+
+        eth_resp = request_get(
+            'https://api.blockcypher.com/v1/eth/main'
+        )
+        if 'height' not in eth_resp:
+            raise ValueError('Failed to query blockcypher for heighest block')
```

Comment:
Here don't raise an exception. If for some reason blockcypher is unavailable this will crash rotkehlchen. Instead return `None`. And where the function is called check for the `None` value and treat it as a no-response and revert back to using etherscan.